R/utils-shared.R

```diff
@@ -500,12 +512,6 @@ ApprovalBarYBottom <- function(lim, ylog, reverse) {
 #' @param ratio A scaling ratio to adjust top or bottom of approval bar rectangle.
 #' @return Approval bar, top or bottom y-axis point, in world coordinates.
 ApprovalBarY <- function(lim, ylog = NULL, reverse, ratio) {
-  if (is.null(ylog)) {
-    # presume the semantics of NULL as FALSE, which may or not be correct, but 
-    # prevents the code from terminating here
-    ylog <- FALSE
-  }
-  
```

Comment:
since you deleted this - you need to add `isEmptyOrBlank(ylog) && ylog` to the if statement below. That's likely why tests are failing. While you're at it, you might as well add `isEmptyOrBlank(reverse) && reverse` instead of just `reverse` to both logic statements.